vscode/src/qirGeneration.ts

```diff
@@ -22,49 +22,52 @@ export class QirGenerationError extends Error {
 }
 
 export async function getQirForActiveWindow(
-  supports_adaptive?: boolean, // should be true or false when submitting to Azure, undefined when generating QIR
+  targetSupportsAdaptive?: boolean, // should be true or false when submitting to Azure, undefined when generating QIR
 ): Promise<string> {
   let result = "";
   const program = await getActiveProgram();
   if (!program.success) {
     throw new QirGenerationError(program.errorMsg);
   }
-
+  const isLocalQirGeneration = targetSupportsAdaptive === undefined;
   const config = program.programConfig;
   const targetProfile = config.profile;
-  const is_unrestricted = targetProfile === "unrestricted";
-  const is_unsupported_adaptive = targetProfile === "adaptive_rif";
-  const is_base = targetProfile === "base";
+  const isUnrestricted = targetProfile === "unrestricted";
+  const isUnsupportedAdaptiveSubmissionProfile =
+    targetProfile === "adaptive_rif";
+  const targetProfileIsBase = targetProfile === "base";
```

Comment:
nitpick: this variable is the one boolean that doesn't start with "is" now.

Reply:
actually, I guess that's not true... `targetSupportsAdaptive` also doesn't start with "is" so maybe this doesn't matter.